custom_components/tuya_local/devices/holman_wx8_irrigation_controller.yaml

```diff
@@ -124,22 +130,37 @@ secondary_entities:
         mapping:
           - scale: 10
   - entity: binary_sensor
+    name: Battery voltage
```

Comment:
If this is a voltage it should be extracting more than one bit and be a sensor rather than a binary_sensor. If it is a low battery warning flag, then leave the name out so that translated names from the battery class are used. If it is anything else, the class is incorrect, so remove it.

Reply:
This is a flag based on the voltage, i.e. "Voltage is ok"

Reply:
So inverted it would meet the spec of `class: battery` for [binary_sensors](https://www.home-assistant.io/integrations/binary_sensor/).

Reply:
![image](https://github.com/make-all/tuya-local/assets/97080551/32eedd01-f42e-4040-a185-522b0130868b)

Reply:
look's good to me?